machine.go

```diff
@@ -25,6 +25,11 @@ const (
 	errMachineAlreadyRegistered   = Error("machine already registered")
 	errMachineRouteIsNotAvailable = Error("route is not available on machine")
 	errMachineAddressesInvalid    = Error("failed to parse machine addresses")
+	errHostnameTooLong            = Error("Hostname too long")
+)
+
+const (
+	maxHostnameLength = 255
```

Comment:
Actually, I just thought, is it the _entire_ url that has the max? Or the segments between . ?

I just had a memory about some kubernetes namespace stuff.

This is mostly to raise this, I'm happy about it as is for now.

Reply:
The Fully Qualified Domain Name has a maximum length of 255 and the labels (segments in between) have a maximum length of 63

Reply:
Ah yes of course